src/BinSkim.Rules/PERules/BA2024.EnableSpectreMitigations.cs

```diff
@@ -241,6 +246,14 @@ public override void AnalyzePortableExecutableAndPdb(BinaryAnalyzerContext conte
                     // mitigations. We do not report here. BA2006 will fire instead.
                     continue;
                 }
+
+                // If the Command Line is not embedded in the binary, then just add a warning for this condition, and there is no point in checking switch states
```

Comment:
>Command Line [](http://example.com/codeflow?start=26&length=12)

what is this strange casing? if this is english text, the casing is wrong. if you referring to a property name like 'RawCommandLine' it should be enquoted and match what you are referring to.

Reply:
Normal casing now.